packages/xpetra/src/Utils/Xpetra_ThyraUtils_def.hpp

```diff
@@ -16,8 +16,6 @@
 
 #include "Xpetra_ThyraUtils_decl.hpp"
 
-namespace Xpetra {
-
```

Comment:
Why do we get rid of the namespace here?

Reply:
Everything below is already qualified with `Xpetra::` so it' not necessary (and confusing) to enclose it in an extra namespace with the same name.